docs/developer-guide/emt-architecture-overview.md

```diff
@@ -36,7 +36,7 @@ outlines the key differences between those.
 | Linux Kernel     | Intel® Kernel 6.12   | Intel® Kernel 6.12                                |
 | Real time        | Available for opt-in | Image variants with standard and RT Kernel provided |
 | Desktop Virtualization | Available      | Dedicated non-RT image variant provided                                    |
-| Add-on packages  | Available for opt-in: Docker + K3s | Built into image: Docker + K3s      |
+| Add-on packages  | Available for opt-in: Docker + K3s | Downloaded during installation: Docker + K3s      |
```

Comment:
Docker is not used as of now in immutable images, only EMT-D it is users choice to install.
k3s and extensions are installed during provisioning.